libsrc/apple2/initcwd.s

```diff
@@ -39,3 +50,9 @@ initcwd:
         sta     __cwd,x
 
 done:   rts
+
+        .rodata
+
+mli_parameters:
+        .byte $01     ; number of parameters
+        .addr __cwd   ; address of parameter
```

Comment:
Sorry for the nitpicking but at least in the Apple II specific code, on-the-same-line comments all start on the same column and begin with an upper case char.

Reply:
Thanks, fixed. I also added an explanation comment before the mli call, for clarity.